### PR TITLE
fix: flaky e2e tests

### PR DESCRIPTION
**Pull request by ethanalvizo**
Fixes:

- code not pasting/typing into editor
- deleting tables between tests
- menus not closing between steps
- using dblclick over drag when possible

Does not fix:
- table disconnect observed on DHC version 0.27.1

### CODE REVIEW

File: tests/utils.ts [hunk 1]
```diff
@@ -120,6 +120,10 @@ export async function pasteInMonaco(
       element.dispatchEvent(clipboardEvent);
     }, text);
   }
+
+  if (text.length) {
```
Comment (mofojed) [inline tests/utils.ts:124]:
```suggestion
  if (text.length > 0) {
```

File: tests/table-operations.spec.ts [hunk 2]
```diff
@@ -105,13 +105,21 @@ async function artificialWait(page: Page, tableNumber = 0) {
   await page.getByTestId('btn-page-close').first().click();
 }
 
+const tableName = generateVarName('t');
 test.beforeEach(async ({ page }) => {
   await page.goto('');
 
+  page.on('console', msg => {
+    if (msg.type() === 'error') console.log(`Error text: "${msg.text()}"`);
+  });
```
Comment (mofojed) [inline tests/table-operations.spec.ts:114]: Do we need to do this? It's already logged in the console and can get it from the browser logs

**Review by mofojed — CHANGES_REQUESTED** [review]

File: tests/table-operations.spec.ts (hunk 2 above)
Comment (ethanalvizo) [reply tests/table-operations.spec.ts:114]: No not necessarily I just found it convenient having it show up in your terminal as it goes through the test as well

Comment (mattrunyon) [reply tests/table-operations.spec.ts:114]: You don't get the browser logs from playwright unless you do this I think

**Review by mofojed — APPROVED** [review]

File: tests/utils.ts [hunk 3]
```diff
@@ -154,6 +154,10 @@ export async function pasteInMonaco(
     // Sanity check the paste happened
     await expect(locator.locator('textarea')).not.toBeEmpty();
   }
+
+  if (text.length > 0) {
+    await expect(locator.locator('textarea')).not.toBeEmpty();
+  }
```
Comment (mofojed) [inline tests/utils.ts:160]: This is duplicated, no need for this block
```suggestion
```

**Review by mofojed — CHANGES_REQUESTED** [review]

**Review by mofojed — APPROVED** [review]

**Pull request merged**